common/build.gradle.kts

```diff
@@ -1,8 +1,12 @@
-plugins {
-    java
-}
+group = "org.jboss.gm.common"
 
-version = "0.1-SNAPSHOT"
+publishing {
+    publications {
+        register("mavenJava", MavenPublication::class) {
```

Comment:
Why is this needed?

Reply:
Trying to get the common module published so that the plugins can use it